src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java

```diff
@@ -236,4 +228,26 @@ private static boolean checkIsAllLinesAreWhitespace(String[] lines, int lineFrom
         }
         return result;
     }
+
+    /**
+     * Calculates the left curly corresponding to the block to be checked.
+     *
+     * @param ast a {@code DetailAST} value
+     * @return the left curly corresponding to the block to be checked
+     */
+    private static DetailAST findLeftCurly(DetailAST ast) {
+        final DetailAST leftCurly;
+        if (ast.getType() == TokenTypes.LITERAL_CASE
+                && ast.getNextSibling() != null
+                && ast.getNextSibling().getFirstChild().getType() == TokenTypes.SLIST) {
+            leftCurly = ast.getNextSibling().getFirstChild();
+        }
+        else if (ast.findFirstToken(TokenTypes.SLIST) == null) {
```

Comment:
We use the value of this in the `else` statement.
Can we assign this result to a variable and use it in both areas instead of calling it twice?